urllib3/util/timeout.py

```diff
@@ -111,29 +111,32 @@ def _validate_timeout(cls, value, name):
         :param name: The name of the timeout attribute to validate. This is
             used to specify in error messages.
         :return: The validated and casted version of the given value.
-        :raises ValueError: If the type is not an integer or a float, or if it
-            is a numeric value less than zero.
+        :raises ValueError: If the type is not an integer, a float, None, or
+            if it is a numeric value less than or equal to zero.
```

Comment:
Maybe `If the value has a numeric value less than or equal to zero, or if it is not of type integer, float, or NoneType`.